### DISCUSSION TITLE
SO 2.3.51 Import Install NOT Showing PCAP Links

### DISCUSSION THREAD

**Discussion by CyberRoFlo** [Security-Onion-Solutions/securityonion#4305] [Unsupported Versions] [answered]
Hello all! I'm new to SO and just installed an IMPORT version on my laptop using VMWare Fusion.  I followed along with https://www.youtube.com/watch?v=TZ96aBEVhFU&t=491s, and all is fine, except I don't have the same PCAP links next to my alerts like it shows in the video.  Could someone provide some guidance? Thanks!

**Comment by cyberkimh** [comment, marked as answer]
Yeah we've never had those either. I presumed that functionality has been removed. There's a few minor differences from the initial video as we've had quite a few updates since then.

**Reply by jertel** [reply]
Yes, the PCAP link is now under the Quick Action menu. Left-click on a data field (not the timestamp field) and it will pop up a menu with an "Actions" submenu containing the PCAP.

![image](https://user-images.githubusercontent.com/16639611/119835654-05381000-becf-11eb-9cb8-4b577c1732dd.png)
